### PR TITLE
Added gridly@1.1.0

### PR DESCRIPTION
**Pull request by IonicaBizau**
Added [`gridly`](https://github.com/IonicaBizau/gridly)––the 157B grid system. :dizzy: 
Fixes https://github.com/IonicaBizau/gridly/issues/1.

### CODE REVIEW

File: ajax/libs/gridly/package.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    "modern",
+    "browsers"
+  ],
+  "author": "Ionică Bizău <bizauionica@gmail.com> (http://ionicabizau.net)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/IonicaBizau/gridly/issues"
+  },
+  "homepage": "https://github.com/IonicaBizau/gridly#readme",
+  "blah": {
```
Comment (PeterDaveHello) [inline ajax/libs/gridly/package.json]: We should remove this `blah` which has no meaning for cdnjs.

File: ajax/libs/gridly/package.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  "npmName": "gridly",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/gridly-col-widths.min.css"
```
Comment (PeterDaveHello) [inline ajax/libs/gridly/package.json]: There is no need to use a leading slash for the files under the root.

Comment (IonicaBizau) [inline ajax/libs/gridly/package.json]: @PeterDaveHello Recommitted. :smile:

File: ajax/libs/gridly/package.json (hunk 1 above)
Comment (IonicaBizau) [inline ajax/libs/gridly/package.json]: Haha, hoped you will not notice that! I guess this should be part of the unit testing, shouldn't it? :joy:

**Comment by IonicaBizau** [comment]
:tada: :rocket: Thanks!

**Comment by PeterDaveHello** [comment]
Now you can access it via:
https://cdnjs.com/libraries/gridly
https://cdnjs.cloudflare.com/ajax/libs/gridly/1.1.0/gridly-col-widths.min.css

**Pull request merged**